src/main/groovy/liquibase/harness/data/ChangeDataTests.groovy

```diff
@@ -53,41 +61,52 @@ class ChangeDataTests extends Specification {
                 return //sql is right. Nothing more to test
             }
         }
-
+        //TODO: Replace connection workaround for mysql and mariadb
```

Comment:
plz create ticket for those improvements and add ticket number in TODO statement

Reply:
+